src/utils/isPlainObject.js

```diff
@@ -1,6 +1,6 @@
 /**
- * @param {any} obj The object to inspect.
- * @returns {boolean} True if the argument appears to be a plain object.
+ * @param {*} obj The object to inspect.
+ * @returns {Boolean} True if the argument appears to be a plain object.
```

Comment:
Why change this? `boolean` is lowercase in Google Closure types reference (one of JSDoc type “providers”), it is also lowercase in Flow.